code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -476,14 +476,17 @@
 			if(-49 to -11)
 				descriptor = "small"
 			if(-10 to 10)
-				descriptor = "about average height"
+				descriptor = "average height"
 			if(11 to 50)
 				descriptor = "tall"
 			if(51 to 100)
 				descriptor = "huge"
 			else
 				descriptor = "gargantuan"
-		heightString = "[get_pronoun("He")] look[get_pronoun("end")] [descriptor] for \a [species.name]"
+		if(species.name == SPECIES_IPC_SHELL || species.name == SPECIES_IPC_SHELL_ROGUE)
```

Comment:
Don't hardcode this. Add a variable of some sort to the species (set to a bodytype like BODYTYPE_HUMAN) that marks it as "lookalike" or something. If that is set, just output the bodytype below instead of "Human".